### PR TITLE
For review: add proposal for raising uopt for certain stablecoins.

### PR DESCRIPTION
**Pull request by reem**

### CODE REVIEW

File: diffs/AaveV3Ethereum_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_before_AaveV3Ethereum_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_after.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+## Reserve changes
+
+### Reserve altered
```
Comment (katiebk) [inline diffs/AaveV3Ethereum_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_before_AaveV3Ethereum_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_after.md:3]: missing DAI here I think

**Review by katiebk — CHANGES_REQUESTED** [review]
I think you're missing DAI for v3 Ethereum but that's all

File: diffs/AaveV3Ethereum_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_before_AaveV3Ethereum_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_after.md (hunk 1 above)
Comment (reem) [reply diffs/AaveV3Ethereum_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_before_AaveV3Ethereum_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_after.md:3]: updated, removed from hackmd as this param was already set

**Review by katiebk — APPROVED** [review]

File: src/20231116_Multi_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments/RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments.md [hunk 2]
```diff
@@ -15,7 +15,6 @@ A proposal to make RF and Uopt changes to stablecoins on all V3 deployments and
 | Market       | Asset | Parameter | Current Value | Recommended Value |
```
Comment (katiebk) [inline src/20231116_Multi_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments/RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments.md:15]: also base v3 USDbC is still missing here

**Review by katiebk — CHANGES_REQUESTED** [review]

File: diffs/AaveV3Optimism_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_before_AaveV3Optimism_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_after.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+| interestRate | ![before](/.assets/8d9de32bf30b1c9dcf71f07a13b228c69a71a4ce.svg) | ![after](/.assets/577360009ed79d63299cf66d8c80048372f3e17b.svg) |
+
+#### DAI ([0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1](https://explorer.optimism.io/address/0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0xA9F3C3caE095527061e6d270DBE163693e6fda9D](https://explorer.optimism.io/address/0xA9F3C3caE095527061e6d270DBE163693e6fda9D) | [0x6D6D3b7FC50999bf20dE5CC8e0F63AFD18B95f0e](https://explorer.optimism.io/address/0x6D6D3b7FC50999bf20dE5CC8e0F63AFD18B95f0e) |
+| optimalUsageRatio | 80 % | 90 % |
+| maxExcessUsageRatio | 20 % | 10 % |
+| variableRateSlope1 | 4 % | 0 % |
```
Comment (ndelzingaro88) [inline diffs/AaveV3Optimism_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_before_AaveV3Optimism_RaiseUoptTo90ForUSDCUSDTDAIFRAXWhereApplicableAcrossAllV3Deployments_20231116_after.md:21]: Variablerateslope1 and basestableborrowrate should not be modified
cc: @reem

**Review by ndelzingaro88 — COMMENTED** [review]
Added some comments to the PR

**Pull request closed without merging**